addons/web/static/src/views/fields/field.js

```diff
@@ -112,15 +112,14 @@ export class Field extends Component {
 
     get fieldComponentProps() {
         const record = this.props.record;
-        const evalContext = record.evalContext;
 
         let readonlyFromModifiers = false;
         let propsFromNode = {};
         if (this.props.fieldInfo) {
             let fieldInfo = this.props.fieldInfo;
 
             const modifiers = fieldInfo.modifiers || {};
-            readonlyFromModifiers = evalDomain(modifiers.readonly, evalContext);
+            readonlyFromModifiers = record.isReadonly(this.props.name, modifiers);
```

Comment:
the changes in this file are linked to the change in relational model -> we should not do that